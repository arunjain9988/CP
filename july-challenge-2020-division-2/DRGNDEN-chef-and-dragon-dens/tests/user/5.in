15 40
1 3 2 4 3 5 2 3 2 4 3 2 3 1 2
1 2 3 4 5 6 7 8 9 10 11 12 13 14 15
2 1 4
2 4 6
2 6 10
2 8 8
2 7 10
2 9 10
2 11 13
2 5 10
2 5 6
2 8 9
2 1 14
2 9 13
2 1 5
2 2 6
2 1 6
2 13 15
2 14 15
2 15 15
2 1 1
2 8 10
2 4 1
2 6 4
2 10 6
2 8 8
2 10 7
2 10 9
2 13 11
2 10 5
2 6 5
2 9 8
2 14 1
2 13 9
2 5 1
2 6 2
2 6 1
2 15 13
2 15 14
2 15 15
2 1 1
2 10 8