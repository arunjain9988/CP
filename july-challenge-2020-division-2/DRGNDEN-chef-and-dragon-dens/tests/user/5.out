-1
-1
16
8
-1
-1
-1
-1
-1
17
-1
-1
-1
-1
-1
28
-1
15
1
-1
7
10
-1
8
25
19
-1
-1
11
-1
-1
-1
-1
12
13
-1
29
15
1
18